ra_aid/__main__.py

```
@@ -547,79 +565,152 @@ def log_level_type(value):
         type=str,
         help="Set the default model to use for future runs",
     )
+
+    # --- Script Subcommands ---
+
+    # last-cost
+    parser_last_cost = subparsers.add_parser("last-cost", help="Display cost and token usage for the latest session.")
```

Ruff (0.11.9)

Local variable `parser_last_cost` is assigned to but never used

Remove assignment to unused variable `parser_last_cost`

(F841)